server_files/server-setup-config.yaml

```diff
@@ -0,0 +1,324 @@
+      - 407206 #Chatheads
+      - 397217 #chesttracker
+      - 392805 #cleancut
+      - 308793 #ClearDespawn
+      - 405262 #clickthrough
+      - 317590 #Client MAmbience
+      - 547692 #Client Tweaks
+      - 531351 #continuity
+      - 297038 #CraftPresence
+      - 619718 #CullLessLeaves
+      - 506898 #DetailArmorBar
+      - 401978 #Disable Custom Worlds Advice
+      - 515172 #Drippy Loading
+      - 448311 #DripSounds
+      - 335493 #Dynamic FPS
+      - 331965 #enchantment tooltips
+      - 250419 #enct descriptions
+      - 537653 #End Goblin Traders
+      - 446173 #enhanced_attack_indicator
+      - 448233 #Entity Culling
+      - 541329 #EquipmentCompare
+      - 447673 #Extra sodium
+      - 417510 #fallingleaves
+      - 416796 #Fancy Menu
+      - 489631 #fuelinfo
+      - 313219 #HealthOverlay
+      - 271492 #IKWID
+      - 292908 #illuminations
+      - 459496 #indium
+      - 495267 #Inventory Profiles Next
+      - 357540 #InventoryHUD
+      - 455508 #Iris
+      - 541604 #Item borders
+      - 242064 #Item Scroller
+      - 495059 #Just Enough Keys
+      - 416797 #Konkrete
+      - 400322 #lambdabettergrass
+      - 393442 #lambdynamiclights
+      - 433518 #lazydfu
+      - 542478 #Legendary Tooltips
+      - 325492 #Light Overlay
+      - 545836 #MiniEffects
+      - 308702 #Mod Menu
+      - 435141 #No Angled brackets
+      - 530816 #no telemetary
+      - 296468 #NoFog
+      - 388252 #Notes
+      - 550387 #otter
+      - 618770 #paginatedadvancements
+      - 547474 #PickUpNotifier
+      - 365521 #Pling
+      - 407981 #Precise Block Placement
+      - 334259 #Presence Footsteps
+      - 463169 #Puddles Bugged
+      - 511319 #Reeses Sodium Options
+      - 415758 #Smooth Boot
+      - 394468 #Sodium
+      - 411557 #tooltipfix
+      - 245733 #torohealth
+      - 521126 #visuality
+      - 349630 #WI Zoom
+      - 547358 #wmitaf
+
+  # The base path where the server should be installed to, ~ for current path
+  baseInstallPath: server/
+
+  # a list of files which are supposed to be ignored when installing it from the client files
+  # this can either use regex or glob {default glob: https://docs.oracle.com/javase/8/docs/api/java/nio/file/FileSystem.html#getPathMatcher-java.lang.String-}
+  # specify with regex:.... or glob:.... if you want to force a matching type
+  ignoreFiles:
+    - mods/Overrides.txt
+    - mods/optifine*.jar
+    - mods/optiforge*.jar
+    - resources/**
+    - shaderpacks/**
+    - kubejs/client_scripts/**
+    - packmenu/**
+    - openloader/resources/**
+    - config/fancymenu/**
+
+  # often a server needs more files, which are nearly useless on the client, such as tickprofiler
+  # This is a list of files, each ' - ' is a new file:
+  # url is the directlink to the file, destination is the path to where the file should be copied to
+  additionalFiles:
+    - url: https://mediafiles.forgecdn.net/files/3682/488/krypton-0.1.9.jar
+      destination: mods/krypton-0.1.9.jar
```

Comment:
I added krypton as you suggested, but it's already in the client files. I guess you can remove it from the client now?